.github/workflows/ci_pipeline.yml

```diff
@@ -156,7 +228,7 @@ jobs:
           echo "print('Loading sitecustomize.py...')
           import coverage
           coverage.process_startup() " > sitecustomize.py
-          coverage run -m pytest --continue-on-collection-errors -v --durations=0 pySDC/tests -m monodomain
+          coverage run -m pytest --continue-on-collection-errors -v --durations=0 pySDC/projects/Monodomain/tests
```

Comment:
Also here I'd like to keep the marker

Reply:
Same as above. I chose to remove the marker, as the path limits the tests already thematically. Therefore, the marker was an additional potential point of failure (skipping tests instead of running them). But also here: We/I can also include that marker again.

Reply:
Readded the marker in b858c33c7fdec93b049845f5232f5b817c127c27